io_scene_godot/__init__.py

```diff
@@ -39,6 +39,83 @@
 }
 
 
+class GodotTextProps(bpy.types.Panel):
```

Comment:
Could you move this into another pull request? thanks

Also probably better to put into another script something like `ui_components.py`, there will be other thing we can do to have more UI support in the future.

Reply:
fixed